sonar/common/jsonschemas/validation-v1.0.0.json

```diff
@@ -0,0 +1,124 @@
+{
+  "title": "Validation",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "status": {
+      "title": "Status",
+      "type": "string",
+      "default": "in_progress",
+      "enum": [
+        "in_progress",
+        "to_validate",
+        "validated",
+        "rejected",
+        "ask_for_changes"
+      ]
+    },
+    "action": {
+      "title": "Action",
+      "type": "string",
+      "default": "save",
+      "enum": [
+        "save",
+        "publish",
+        "approve",
+        "reject",
+        "ask_for_changes"
+      ]
+    },
+    "user": {
+      "title": "User",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "User",
+          "type": "string",
+          "pattern": "^https://sonar.ch/api/users/.*?$"
+        }
+      }
+    },
+    "comment": {
+      "title": "Comment",
+      "type": "string",
+      "minLength": 1
+    },
+    "logs": {
+      "title": "Logs",
+      "type": "array",
+      "items": {
+        "title": "Log",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "status": {
```

Comment:
This field seems used twice. It is a candidate to a definition. Is it right to have two statuses? Same for actions?